testkitbackend/requests.py

```diff
@@ -90,19 +93,80 @@ def SessionBeginTransaction(backend, data):
     metadata = data.get('txMeta', None)
     timeout = data.get('timeout', None)
     if timeout:
-        timeout = int(timeout)
+        timeout = float(timeout) / 1000
     tx = session.begin_transaction(metadata=metadata, timeout=timeout)
     key = backend.next_key()
     backend.transactions[key] = tx
     backend.send_response("Transaction", {"id": key})
 
 
+def SessionReadTransaction(backend, data):
```

Comment:
This is non-standard casing for Python functions.

Reply:
It's a bit of laziness... the requests in testkit are mapped directly to function names, didn't have the energy to change casing of the requests